README.md

```diff
@@ -747,6 +748,39 @@ pipeline.notifyOnFailure(PipelineNotification.chime({
 }));
 ```
 
+## ECR Mirror
+
+Builds commonly use Docker images, and these typically come from DockerHub. In fact, delivlib defaults its build
```

Comment:
```suggestion
Builds commonly used Docker images, and these typically come from DockerHub. In fact, delivlib defaults its build
```

Reply:
It should be a statement in the present, and not in the past.

I'm going to adjust it this way instead.

"Builds commonly use Docker images from DockerHub as their base image."